Gem/Code/Source/Platform/Windows/HighInstanceTestComponent_Traits_Platform.h

```diff
@@ -6,6 +6,6 @@
  *
  */
 
-#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_X       22
-#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_Y       22
-#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_Z       22
+#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_X       100
+#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_Y       100
+#define ATOMSAMPLEVIEWER_TRAIT_HIGH_INSTANCE_COUNT_TEST_COMPONENT_LATTICE_SIZE_Z       10
```

Comment:
My plan for android and other platforms is that we would have specific performance tests for them. The current tests are for the 2022 goals on windows desktops. With that in mind I don't think palifying the test parameters is useful.

Reply:
I agree that this test is not suitable for android. Palifying is to avoid having android frozen in this sample or even crash for any resources out of limit. In fact, some other samples have the same issue.
I think the eventual scenario is to have android running a different set of samples, but this is not implemented. For now, we can just use the macros.